tests/test_detector.py

```diff
@@ -130,8 +132,8 @@ def test_detector_set_field_float_numbers():
 def test_detector_set_field_missing_values():
     detector = Detector(field_missing_values=["", "67"])
     assert detector.field_missing_values == ["", "67"]
-    detector.field_missing_values = ["", "70"]
```

Comment:
I think it's better to keep any existent test assertions to be sure that we don't introduce any breaking changes